setup.py

```diff
@@ -1,28 +1,77 @@
+import os
+import os.path
+
 try:
     from setuptools import setup
 except ImportError:
     from distutils.core import setup
 
-long_description = ""
-with open('README.rst') as f:
-    long_description = f.read()
 
-setup(
-    name='ImageHash',
-    version='3.0',
+NAME = 'imagehash'
+DESCRIPTION = 'Image Hashing Library'
+DIR = os.path.abspath(os.path.dirname(__file__))
+
+
+package_data = {
+    NAME: [os.path.join('tests', 'data', '*')],
+}
```

Comment:
Package the tests with the code, so that users can also test the deployed code base